automation/jmeter/runBenchmarkTest.sh

```diff
@@ -2,15 +2,18 @@
 
 function runBenchmark(){
 local HOST_ADDRESS=$1
-printf "Running jmeter test and generating JTL file."
 
+printf "Running jmeter test and generating JTL file."
+JVM_ARGS="-Xms4g -Xmx6g"
 jmeter -n -t "${currentDir}/AMQ_Jmeter_Test_Plan_local.jmx" \
   -JproducerDuration=10 \
   -JconsumerDuration=20 \
+  -JproducerSamples=1000 \
+  -JconsumerSamples=2000 \
   -l "${outputDir}/jmeter_results_${HOST_ADDRESS}.jtl"
 
 printf  "Calculating throughput from JTL file."
-./gen_throughput.sh "-i" "${outputDir}/jmeter_results_${HOST_ADDRESS}.jtl" -a 200 -b 200 -f csv -c "JMS Subscriber" -p "JMS Publisher" > "${outputDir}/jmeter_csv_results_${HOST_ADDRESS}.csv"
+./gen_throughput.sh "-i" "${outputDir}/jmeter_results_${HOST_ADDRESS}.jtl" -a 1000 -b 2000 -f csv -c "JMS Subscriber" -p "JMS Publisher" > "${outputDir}/jmeter_csv_results_${HOST_ADDRESS}.csv"
```

Comment:
`-a` is for consumer sample size and it's value is `2000` in the jmeter jmx file but it's specified as `1000` here.
Same goes for `-b` (producer sample size) - value in jmx is `500` but it is specified as `2000` here.

Reply:
Thanks for pointing out. I have updated the PR.